plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/callables/FunctionSignature.kt

```diff
@@ -24,6 +25,6 @@ interface FunctionSignature : CallableSignature {
         get() = receiver?.type
     override val paramTypes: List<TypeEmbedding>
         get() = params.map { it.type }
-}
 
-fun FunctionSignature.parametersByFirSymbols(): Map<FirBasedSymbol<*>, FirVariableEmbedding> = params.associateBy { it.symbol }
+    fun primaryConstructorInvariants(returnVariable: VariableEmbedding): List<ExpEmbedding>
```

Comment:
Sorry, missed this earlier.  Does this have to be a function of this interface?  It seems like a fairly specialised case that doesn't really fit with `FunctionSignature` in general.

Reply:
Oh, I just changed a comment here to clarify what is going on and why this test is interesting. Anyway this is a new test (as well as inheritance_fields.kt) and maybe we don't need it actually (but there're no other tests where we define a collection so why not)